.github/docs/openapi3gen_pack.txt

```diff
@@ -0,0 +1,9 @@
+package pack // import "github.com/getkin/kin-openapi/openapi3gen/pack"
```

Comment:
Ah we don't want to include this in the docs but you can't export a struct from a test package